website/docs/targets.md

```diff
@@ -22,7 +22,7 @@ azlinux3/worker                  Builds the base worker image responsible for bu
 debug/gomods                     Outputs all the gomodule dependencies for the spec
 debug/resolve                    Outputs the resolved dalec spec file with build args applied.
 debug/sources                    Outputs all sources from a dalec spec file.
-mariner2/container (default)     Builds a container image for
+mariner2/container (default)     Builds a container image with mariner2 as the base image.
```

Comment:
which one is the default?

Reply:
> which one is the default?

Its default for `path.Dir(target)`.
So when `--target=mariner2`, the `mariner2/container` target is what gets executed.

Reply:
Oh I'm realizing this is confusingly worded, it should be "default for distro" or "default for platform". I think I'm going to move the high-level targets `mariner2`, `azlinux3` and `windowscross` out as their items and mark them as aliases.